core/src/main/java/org/infinispan/transaction/impl/TransactionTable.java

```diff
@@ -606,7 +607,8 @@ private void shutDownGracefully() {
          log.unfinishedTransactionsRemain(localTransactions == null ? 0 : localTransactions.size(),
                                           remoteTransactions == null ? 0 : remoteTransactions.size());
          if (trace) log.tracef("Unfinished local transactions: %s",
-                    localTransactions.values().stream().map(AbstractCacheTransaction::getGlobalTransaction));
+            localTransactions.values().stream().map(tx -> tx.getGlobalTransaction().toString())
+               .collect(Collectors.joining(", ")));
```

Comment:
I've noticed that the output is slightly different from the line below, because `Collection.toString` puts everything the element in square brackets. Could you change it to use `Collectors.toList()` instead?